src/js/socket.js

```diff
@@ -0,0 +1,20 @@
+import { io } from 'socket.io-client';
+
+export const socket = io.connect('http://localhost:5000');
```

Comment:
we will need to update this to use the env. The original code used:

import.meta.env.VITE_API_BASE_URL

Reply:
Unsure what you mean by this, my .env holds:
http://localhost:5000/